#5 Support branch name input (#6)
Eomm/massive-wax: README.md

  -b, --branch=branch                  [default: wax] the branch name where apply the changes

>   -M, --main-branch=main-branch        [default: main] the main branch in the origin repo

Reviewer: I would keep `master` as default just to release this PR as semver minor

```suggestion
  -O, --pr-origin=master        [default: master] the target PR branch name in the origin repository
```

  - clone the repos in your local env
  - process all the files of the cloned repos
  - commit the changes in a dedicated branch

>   - open a PR to the main branch in the origin repo

Reviewer: ```suggestion
  - open a PR to the `pr-origin` branch in the origin repo
```

  All these steps are optionals.

EXAMPLES
  Change all the LICENCE file of your org

>     $ upgrade -K GITHUB-TOKEN --fork -p='./toMit.js' -t='Changed license' -c='chore changed license' -M main -b licensebranch -r 

Reviewer: ```suggestion
    $ upgrade -K GITHUB-TOKEN --fork -p='./toMit.js' -t='Changed license' -c='chore changed license' -O main -b licensebranch -r 
```
